src/science/canopy/cable_canopy.F90

```diff
@@ -65,6 +63,10 @@ SUBROUTINE define_canopy(bal,rad,rough,air,met,dels,ssnow,soil,veg, canopy,clima
 USE cable_math_constants_mod,  ONLY : CPI_C  => PI
 USE cable_other_constants_mod, ONLY : CLAI_THRESH  => LAI_THRESH
 
+    USE cable_def_types_mod
```

Comment:
all these USEs are just re-ordered to fit into the format:
`
! subs
USE...
!data
USE ...
`